src/seedu/addressbook/data/person/Phone.java

```diff
@@ -56,4 +56,8 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    //implement printable
+    public static Printable getPrintableString( Printable EXAMPLE) {
+		return EXAMPLE;
```

Comment:
You are returning a static value. You must return the value assigned when the instance of a Phone is created.